plan/plan.go

```diff
@@ -164,8 +166,14 @@ type PhysicalPlan interface {
 	json.Marshaler
 	Plan
 
-	// matchProperty means that this physical plan will try to return the best plan that matches the required property.
-	// childPlanInfo means the plan infos returned by children.
+	// matchProperty calculates the cost of the physical plan if it matches the required property.
+	// It's usually called at the end of convert2PhysicalPlan, some PhysicalPlan does not implement it as there is
+	// no property to match, they just do the cost calculation directly.
+	// If not match, the cost will be set to MaxInt64, so it will not be chosen as the result physical plan.
+	// childrenPlanInfo are used to calculate the result cost of the plan.
+	// The return *physicalPlanInfo will be chosen as final plan if it has the lowest cost.
```

Comment:
The return *physicalPlanInfo will be chosen as final plan if it has the lowest cost. 
->
The returned *physicalPlanInfo will be chosen as the final plan if it has the lowest cost.